addons/binding/org.openhab.binding.minecraft/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,193 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="minecraft" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<bridge-type id="server">
+		<label>Minecraft Bukkit Server</label>
+		<description>Server from which data is being fetched.</description>
+
+		<channels>
+			<channel id="name" typeId="name" />
+			<channel id="online" typeId="online" />
+			<channel id="bukkitVersion" typeId="bukkitVersion" />
+			<channel id="version" typeId="version" />
+			<channel id="players" typeId="players" />
+			<channel id="maxPlayers" typeId="maxPlayers" />
+		</channels>
+
+		<config-description>
+			<parameter name="hostname" type="text" required="true">
+				<label>Hostname or IP</label>
+				<context>network-address</context>
+				<description>Hostname or IP of the server.</description>
+			</parameter>
+
+			<parameter name="port" type="integer">
+				<label>Port</label>
+				<description>The port on which the server can be accessed.</description>
+				<default>10692</default>
+			</parameter>
+		</config-description>
+	</bridge-type>
+
+	<thing-type id="player">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="server" />
+		</supported-bridge-type-refs>
+
+		<label>Player</label>
+		<description>A connected player connected to the Minecraft server.</description>
+
+		<channels>
+			<channel id="playerName" typeId="playerName" />
+			<channel id="playerOnline" typeId="online">
+				<label>Online</label>
+				<description>Online status of the player.</description>
+			</channel>
+			<channel id="playerLevel" typeId="playerLevel" />
+			<channel id="playerTotalExperience" typeId="playerTotalExperience" />
+			<channel id="playerExperiencePercentage" typeId="playerExperiencePercentage" />
+			<channel id="playerHealth" typeId="playerHealth" />
+			<channel id="playerWalkSpeed" typeId="playerWalkSpeed" />
+			<channel id="playerLocation" typeId="location">
+				<label>Location</label>
```

Comment:
you do not use the location channel type anywhere else, so it does not seem to be necessary to override label and desc here.